content/pl/processes.md

```diff
@@ -0,0 +1,15 @@
+## VI. Procesy
+### Odpal aplikację jako jeden lub więcej bezstanowych procesów
+
+Aplikacja jest uruchamiana w środowisku wykonawczym jako jeden lub więcej *procesów*.
+
+W najprostszym przypadku, kod aplikacji jest samodzielnym skryptem, środowiskiem wykonawczym jest laptop developera z wsparciem dla języka programowania a proces jest uruchamiany za pomocą linii komend (na przykład `python my_script.py`). W zupełnie innym przypadku wdrożenie produkcyjne mocno rozwiniętej aplikacji może wymagać wiele [process types, instantiated into zero or more running processes](./concurrency).
+
+**Wg dwunastu aspektów, procesy są bezstanowe i [nie-współdzielące](http://en.wikipedia.org/wiki/Shared_nothing_architecture).**  Jakiekolwiek dane, które potrzebują zapisu, muszą zostać zmagazynowane w stanowej [usłudze wspierającej](./backing-services), zazwyczaj będącej bazą danych.
+
+Miejsce w pamięci lub system plików procesu mogą być używane jako tymczasowy cache dla pojedynczych operacji. Przykładem jest pobieranie dużych plików, działanie na nich, a następnie zapisywanie wyników operacji w bazie danych. Aplikacja dwunastu aspektów nigdy nie zakłada, że jakiś fragment informacji zapisany w pamięci lub dysku będzie dostępny w przyszłości podczas jakiegokolwiek zapytania -- wraz z wieloma aktywnymi procesami rośnie szansa, że przyszłe zapytanie zostanie obsłużone przez zupełnie inny proces. Nawet w przypadku pojedynczego procesu, restart (spowodowany przez deployment kodu, zmianę konfiguracji lub środowisko wykonawcze relokujące proces na inną fizyczną lokalizację) zazwyczaj usunie wszystkie dane z lokalnego stanu aplikacji (system plików, pamięć podręczna).
+
+Narzędzie do pakowania plików, z których korzysta aplikacja (takie jak [Jammit](http://documentcloud.github.com/jammit/) lub [django-compressor](http://django-compressor.readthedocs.org/)) używają systemu plików jako cache dla skomilowanych zasobów.  Wg dwunastu aspektów taka kompilacja powinna mieć miejsce podczas [etapu tworzenia builda](./build-release-run), jak to się dzieje np. w [Rails asset pipeline](http://guides.rubyonrails.org/asset_pipeline.html), niż podczas uruchamiania aplikacji.
+
+Niektóre systemy sieciowe polegają na tzw. ["sticky sessions"](http://en.wikipedia.org/wiki/Load_balancing_%28computing%29#Persistence) -- to znaczy, że sesja użytkownika jest cachowana w pamięci procesu aplikacji, oczekując, że kolejne zapytania dotyczące użytkownika będą kierowane do tego samego procesu. "Sticky sessions" są pogwałceniem dwunastu aspektów aplikacji i nigdy nie powinny być używane jako źródło informacji. Dane sesyjne są dobrym kandydatem dla magazynu danych oferującego wygasanie danych w czasie, jak np. [Memcached](http://memcached.org/) czy [Redis](http://redis.io/).
```

Comment:
1. `to znaczy` -> `oznacza to`
2. `oczekując` -> `zakładając`
3. `Dane sesyjne` -> `Dane sesji` 
4. `są dobrym kandydatem` -> `nadaja się` (po polsku `bycie kandydatem tutaj zdecydowanie nie pasuje`